.github/workflows/oblt-github-commands.yml

```diff
@@ -50,8 +50,7 @@ jobs:
 
               Just comment with:
               - \`/oblt-deploy\` : Deploy a Kibana instance using the Observability test environments.
-              - \`run\` \`elasticsearch-ci/docs\` : Re-trigger the docs validation. (use unformatted text in the comment!)
-
+              - \`run\` \`docs-build\` : Re-trigger the docs validation. (use unformatted text in the comment!)
```

Comment:
@v1v Is this two separate commands? As in, there is `run` and then there is `docs-build`? If not I suggest it formatted as a single command `run docs-build`

Reply:
> Is this two separate commands or one?

one, `run docs-build` is the command.

If the message is not clear please let me know and we can change the content to be more accurate. I split with special chars so it avoids the recurrence of being detected as `run docs-build` by the CI docs pipeline